src/main/java/emissary/output/DropOffUtil.java

```diff
@@ -973,6 +962,15 @@ public Date getEventDate(final IBaseDataObject d, @Nullable final IBaseDataObjec
         return eventDate;
     }
 
+    /**
+     * Getter for dateParser
+     * 
+     * @return the date parser to use
+     */
+    public String getDateParser() {
```

Comment:
Not sure I understand the return type of this method being a string. If I want a date parser, then I'm going to get something like `LEGACY`, so what do I do with it? Is there a method somewhere to translate that to an actual date parser?